software/tracksight/backend/app/influx_handler.py

```diff
@@ -0,0 +1,117 @@
+from dateutil.parser import parse
+from datetime import timedelta
+import requests
+
+URL = "https://us-east-1-1.aws.cloud2.influxdata.com"
+BUCKET = "testing"
+
+TEMP_TOKEN = "pyh_P66tpmkqnfB6IL73p1GVSyiSK_o5_fmt-1KhZ8eYu_WVoyUMddNsHDlozlstS8gZ0WVyuycQtQOCKIIWJQ=="
```

Comment:
Going forward we'll need to do a few things:

1. Delete this token from grafana
2. Give developers a token that should be used for development, but not visible from the codebase
3. Create a new token and have it be an env variable in the deployed version